README.md

```diff
@@ -64,10 +82,10 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+* Hat tip to anyone whose code was used
+* Inspiration
+* etc
```

Comment:
- [ ] As you have used someone else design, it is important to give a credit to him/her as a privilege. This helps anyone who visits your website but is interested in the design you have followed get to know the source 💡. Moreover, it is among the project requirements to give credit to [Cindy Shin](https://www.behance.net/adagio07) the author of the original design you used. Please add her here in the Acknowledgement section and the link to the design. 😃 👍